### PR TITLE
github_actions: Translate .travis.yml to actions

### PR DESCRIPTION
**Pull request by MathieuBordere**
Succeeded here on my repo: https://github.com/MathieuBordere/go-dqlite/pull/2

### CODE REVIEW

File: .github/workflows/build-and-test.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+jobs:
+  build-and-test:
+    runs-on: ubuntu-18.04
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Set up Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: 1.14
```
Comment (stgraber) [inline .github/workflows/build-and-test.yml:20]: Should test similar to go-lxc (1.13.x through 1.16.x)

File: .github/workflows/build-and-test.yml [hunk 2]
```diff
@@ -0,0 +1,44 @@
+name: go-dqlite-build-and-test
```
Comment (stgraber) [inline .github/workflows/build-and-test.yml:1]: A more friendly name would be nice

File: .github/workflows/cla-check.yml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+name: go-dqlite-cla-check
+
+on: [pull_request]
```
Comment (stgraber) [inline .github/workflows/cla-check.yml:3]: I'd still write it as:
```
on:
 - pull_request
```

Just for consistency

File: .github/workflows/cla-check.yml [hunk 4]
```diff
@@ -0,0 +1,10 @@
+name: go-dqlite-cla-check
```
Comment (stgraber) [inline .github/workflows/cla-check.yml:1]: A better name would help

Comment (stgraber) [reply .github/workflows/cla-check.yml:1]: Maybe something like `Canonical CLA`?

**Comment by stgraber** [comment]
Left a few comments. This should be pretty close to what we do for `go-lxc` so maybe take a look at those and try to line things up?

**Comment by MathieuBordere** [comment]
> Left a few comments. This should be pretty close to what we do for `go-lxc` so maybe take a look at those and try to line things up?

Thanks, will do.

**Pull request merged**